artifacts/src/main/resources/issuance/credential-request-message-schema.json

```diff
@@ -24,19 +24,7 @@
         "credentials": {
           "type": "array",
           "items": {
-            "type": "object",
-            "properties": {
-              "format": {
-                "type": "string"
-              },
-              "credentialType": {
-                "type": "string"
-              }
-            },
-            "required": [
-              "credentialType",
-              "format"
-            ]
+            "type": "string"
```

Comment:
This would make credential requests with client-supplied data impossible. #205 suggests to add another property to this object. I'd prefer the `credentials` array to hold objects with at least an `id` or `credentialObjectId` property.

Reply:
adapted in https://github.com/eclipse-dataspace-dcp/decentralized-claims-protocol/pull/215/commits/a7e90f484f50d7894d74ff0c712c331de65d8f56